OracleTuxedo/core/dockerfiles/12.1.3/Dockerfile

```diff
@@ -7,25 +7,25 @@
 #
 
 # Pull base image
-FROM oraclelinux
+FROM container-registry.oracle.com/java/serverjre
 
 MAINTAINER Todd Little <todd.little@oracle.com>
 
 # Core install doesn't include unzip or gcc, add them
 # Create the installation directory tree and user oracle with a password of oracle
-RUN yum -y install unzip gcc file; yum -y clean all && \
+RUN yum -y install unzip gcc file which; yum -y clean all && \
```

Comment:
Please change the `;` to `&&` so that the build will abort if either of the yum commands fail.